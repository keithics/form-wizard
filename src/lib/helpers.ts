export function getParams(): { [p: string]: string } {
  const urlSearchParams = new URLSearchParams(window.location.search);
  return Object.fromEntries(urlSearchParams.entries());
}
