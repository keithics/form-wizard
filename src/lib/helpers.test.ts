import { expect, test } from '@jest/globals';

import { getParams } from './helpers';

describe('Parsing GET queries', () => {
  test('getParams query', () => {
    window.location.search = '?one=1&two=2';

    expect(getParams()).toMatchSnapshot();
  });
});
