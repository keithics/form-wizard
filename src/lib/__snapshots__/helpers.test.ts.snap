// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parsing GET queries getParams query 1`] = `
Object {
  "one": "1",
  "two": "2",
}
`;
